canvas_modules/common-canvas/src/common-properties/components/flexible-table/flexible-table.jsx

```diff
@@ -514,6 +517,18 @@ class FlexibleTable extends React.Component {
 			)
 			: null;
 
+		let tableHeight = 0;
+		const multiSelectEditRowsRem = 2 * REM_HEADER_HEIGHT; // multi-select adds two rows when selectedEditRow
+		const multiSelectEditRowsPixels = multiSelectEditRowsRem * ONE_REM_HEIGHT;
+		if (this.state.rows !== -1 && this.state.tableHeight) {
+			const remHeight = parseInt(this.state.tableHeight.match(/^[0-9]{1,2}/i)[0], 10);
```

Comment:
I think you can remove match and just have `parseInt(this.state.tableHeight, 10);`

Reply:
yeesss!